templates/jobs-header.tmpl

```diff
@@ -2,4 +2,4 @@
 {?which_jobs=completed?:<FORM ACTION="{?printer_name=?/jobs:{printer_uri_supported}}" METHOD="GET"><INPUT TYPE="HIDDEN" NAME="which_jobs" VALUE="completed"><INPUT TYPE="SUBMIT" VALUE="Show Completed Jobs"></FORM>}
 {?which_jobs=all?:<FORM ACTION="{?printer_name=?/jobs:{printer_uri_supported}}" METHOD="GET"><INPUT TYPE="HIDDEN" NAME="which_jobs" VALUE="all"><INPUT TYPE="SUBMIT" VALUE="Show All Jobs"></FORM>}
 
-<P ALIGN="CENTER">{?which_jobs=?Jobs listed in print order; held jobs appear first.:{which_jobs=Jobs listed in ascending order.?:Jobs listed in descending order.}}</P>
+<P ALIGN="CENTER">{?which_jobs=?Active jobs listed in processing order▼; held jobs appear first.:{which_jobs=all?Jobs in plain reception order▼:Completed jobs in order of finish/cancel/abort▲}}</P>
```

Comment:
"in numeric order" instead of "in plain reception order" I think.

"Jobs in order of completion or cancellation"?

Reply:
numeric order is mathematically correct, but what is the measurement? The numeric value is "sequence number of received jobs", right? Core matter is to name the measurant. So, how about "Jobs in sequential order" (all)? I agree on your suggestion "Jobs in order of completion or cancellation" (completed).

Reply:
We show the job ID number, which is the number the "all jobs" is sorted on.